web/text_layer_builder.css

```diff
@@ -96,10 +96,11 @@
   }
 
   ::selection {
+    /* stylelint-disable declaration-block-no-duplicate-properties */
```

Comment:
The updated Stylelint version reports the error on the _first_ occurrence of the duplicate CSS rule now. However, we can't use the `stylelint-disable-next-line` comment within a preprocessor block because something removes the comment markers then; see the following output when running `npx gulp unittest` with that change:

```
[14:33:49] CssSyntaxError in plugin "gulp-postcss"
Message:
    /home/timvandermeij/Documenten/Ontwikkeling/pdf.js/Code/viewer.css:350:36: Unknown word

  348 | 
  349 |   ::selection {
> 350 |        stylelint-disable-next-line declaration-block-no-duplicate-properties 
      |                                    ^
  351 |     background: rgba(0 0 255 / 0.25);
  352 |     background: color-mix(in srgb, AccentColor, transparent 75%);
```

Hence this was the only way I was able to find to appease both Stylelint and the preprocessor.

Reply:
Yes, unfortunately the old pre-processor used for HTML/CSS doesn't handle comments *inside* of pre-processor blocks correctly.
That's something that has annoyed me from time to time, but it's just never felt worthwhile to spend time trying to fix given that it's rarely an issue.